frontend/ui/data/css_tweaks.lua

```diff
@@ -303,12 +303,49 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
                 separator = true,
             },
             {
-                id = "ruby_inline";
-                title = _("Render <ruby> content inline"),
-                description = _("Disable handling of <ruby> tags and render them inline."),
-                css = [[
-ruby { display: inline !important; }
-                ]],
+                title = _("Ruby"),
+                {
+                    title = _("About ruby"),
+                    info_text = _([[
+Ruby characters are small glosses on the Asian languages to show the logographs' (Hanzi, Kanji, Hanja, etc) pronunciation.
+These tweaks allow small stylistic changes and larger ones to make ruby easier to read or ignore.]]),
+                    separator = true,
+                },
+                {
+                    id = "ruby_font_sans_serif";
+                    title = _("Nicer ruby font"),
+                    description = _([[
+Use a sans serif font to display all ruby text for a more 'book-like' feeling.
+Also force the normal text weight when used with lighter or bolder fonts.]]),
```

Comment:
I know CSS calls it `normal` but I'm not sure what I think of that in this context.
```suggestion
Also force the regular text weight when used with lighter or bolder fonts.]]),
```